pkg/operator/sync.go

```diff
@@ -286,11 +286,7 @@ func (optr *Operator) syncRenderConfig(_ *renderConfig) error {
 
 func (optr *Operator) syncCustomResourceDefinitions() error {
 	crds := []string{
-		"manifests/machineconfig.crd.yaml",
 		"manifests/controllerconfig.crd.yaml",
```

Comment:
since you're leaving the controllerconfig here as-is, does that mean it should also be excluded from the move to the api repo?

Reply:
I think so, the CC is an internal sync mechanism for the MCO, nothing users/admins should tweak